deploy/kubernetes/charts/serai/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
What is service account?

Reply:
deactivated service account and deleted yamls